pkg/virtualKubelet/metrics/metrics.go

```diff
@@ -0,0 +1,64 @@
+package metrics
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// Metrics port
+	MetricsPort = ":9090"
+)
+
+var (
+	// ErrorsCounter is the counter of the errors occurred during the reflection.
+	ErrorsCounter *prometheus.CounterVec
+	// ItemsCounter is the counter of the items extracted from the workqueue and correctly processed.
+	ItemsCounter *prometheus.CounterVec
+	// MetricsLabels is the labels used for the metrics.
+	MetricsLabels []string
+)
+
+// Init initializes the metrics.
+func init() {
+	MetricsLabels = []string{"namespace", "reflector_type"}
+
+	ErrorsCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "liqo_virtual_kubelet_reflection_error_counter",
+			Help: "The counter of the transient errors.",
+		},
+		MetricsLabels,
+	)
+
+	ItemsCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "liqo_virtual_kubelet_reflection_item_counter",
+			Help: "The counter of the items extracted from the workqueue and correctly processed.",
+		},
+		MetricsLabels,
+	)
+
+	// Register the metrics to the prometheus registry.
+	prometheus.MustRegister(ItemsCounter)
+	prometheus.MustRegister(ErrorsCounter)
```

Comment:
Move these lines in the setup function to avoid accidental registration of these metrics on package imports